### PR TITLE
fix: broken build for html template file

### PR DESCRIPTION
**Pull request by vr-greycube**
Create bundle.js for building assessment_result_tool template

### CODE REVIEW

**Comment by vr-greycube** [comment]
fixes #26 
@ruchamahabal please review

**Comment by jigneshpshah** [comment]
@ruchamahabal , would be very much helpful if you can please approve

**Comment by jigneshpshah** [comment]
@deepeshgarg007  could you please help get this merged ?

**Comment by jigneshpshah** [comment]
@nabinhait could you please help review & merge a simple PR ! , waiting since 20 days !!

File: education/hooks.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 # include js, css files in header of desk.html
 # app_include_css = "/assets/education/css/education.css"
 # app_include_js = "/assets/education/js/education.js"
+app_include_js = ["education.bundle.js"]
```
Comment (ruchamahabal) [inline education/hooks.py]:
```suggestion
app_include_js = "education.bundle.js"
```

**Pull request merged**